internal/cli/universal_login.go

```diff
@@ -178,7 +178,7 @@ func updateUniversalLoginCmd(cli *cli) *cobra.Command {
 			b := &management.Branding{}
 			isAccentColorSet := len(inputs.AccentColor) > 0
 			isBackgroundColorSet := len(inputs.BackgroundColor) > 0
-			currentHasColors := current.Colors != nil
+			currentHasColors := current.GetColors() != nil
```

Comment:
I'm not sure if this error is solely for a new tenant/tenant that hasn't touched the values (due to this API being a PATCH API so it's difficult to reset the values)

But on first run of `auth0 ul` in my tenant I got the below error, this repeated for LogoURL and FaviconURL, looking at the API calls in the UI a `GET /api/v2/branding` only returned a subset of data `{"colors":{"primary":"#FF4F40"}}` rather than a full object

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x1009f2604]

goroutine 1 [running]:
github.com/auth0/auth0-cli/internal/cli.Execute.func1()
	/Users/ewanharris/dev/git/auth0-cli/internal/cli/root.go:53 +0xb4
panic({0x101050900, 0x1017f6f60})
	/opt/homebrew/Cellar/go/1.20.2/libexec/src/runtime/panic.go:884 +0x1f4
github.com/auth0/auth0-cli/internal/cli.updateUniversalLoginCmd.func1(0x1400001a900?, {0x140001201c0?, 0x2?, 0x2?})
	/Users/ewanharris/dev/git/auth0-cli/internal/cli/universal_login.go:181 +0x1c4
github.com/spf13/cobra.(*Command).execute(0x1400001a900, {0x140001201a0, 0x2, 0x2})
	/Users/ewanharris/dev/git/auth0-cli/vendor/github.com/spf13/cobra/command.go:916 +0x5c4
github.com/spf13/cobra.(*Command).ExecuteC(0x14000292600)
	/Users/ewanharris/dev/git/auth0-cli/vendor/github.com/spf13/cobra/command.go:1044 +0x340
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/ewanharris/dev/git/auth0-cli/vendor/github.com/spf13/cobra/command.go:968
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	/Users/ewanharris/dev/git/auth0-cli/vendor/github.com/spf13/cobra/command.go:961
github.com/auth0/auth0-cli/internal/cli.Execute()
	/Users/ewanharris/dev/git/auth0-cli/internal/cli/root.go:64 +0x1ac
main.main()
	/Users/ewanharris/dev/git/auth0-cli/cmd/auth0/main.go:6 +0x1c
```